frontend/cypress/integration/common/graph_toolbar.ts

```diff
@@ -187,7 +186,7 @@ Then('user does not see graph duration menu', () => {
 
 Then('user sees selected graph duration {string}', (duration: string) => {
   cy.get('button#time_range_duration-toggle')
-    .find('span[class*="pf-v5-c-menu-toggle__text"]')
+    .find('span[class="pf-v5-c-menu-toggle__text"]')
```

Comment:
looks like removing wildcard did not affected test run

Reply:
No, because the CSS class is exact. In that case, I prefer to remove the wildcard to be more precise in the cypress test.